substrafl_examples/get_started/plot_substrafl_sklearn_fedavg.py

```diff
@@ -0,0 +1,523 @@
+"""
+=======================================
+Substrafl Skearn FedAvg on IRIS dataset
+=======================================
```

Comment:
I would add a Flower image on the thumbnail of the sphinx galery and not the performance curve. This would look better :)

Reply:
I'll try but with sphinx, nothing is easy...